<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationDialog</class>
 <widget class="QWidget" name="CalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>系统标定</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="Viewer2D" name="viewer1" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox1">
       <property name="title">
        <string>内参数标定＆手眼标定</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0" columnminimumwidth="0,120">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="1" column="1">
         <widget class="QPushButton" name="button1Auto">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>自动标定</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="button1Start">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>手动标定</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="button1Solve">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>解算参数</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>设备选择</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton1">
             <property name="text">
              <string>相机A</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton2">
             <property name="text">
              <string>相机B</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="button1Capt">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>捕获帧</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>已捕获 0 帧</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>6</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox2">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>激光平面标定</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="button2Auto">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>自动标定</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="button2Capt">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>捕获帧</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="button2Start">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>手动标定</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="button2Solve">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>解算参数</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Viewer2D" name="viewer2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textLog"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Viewer2D</class>
   <extends>QWidget</extends>
   <header>widget/Viewer2D.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button1Start</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>ManualCalibrateStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button1Capt</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>ManualCapture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button1Solve</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>ManualSolve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button1Auto</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>AutoCalibrateStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>757</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button2Auto</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>AutoLaserStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button2Start</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>ManualLaserStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button2Solve</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>ManualLaserSolve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button2Capt</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>ManualLaserCapture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
